src/main/java/com/lpvs/repository/LPVSLicenseRepository.java

```diff
@@ -35,10 +35,7 @@ public interface LPVSLicenseRepository extends JpaRepository<LPVSLicense, Long>
      * @param spdxId The SPDX identifier of the license.
      * @return The latest {@link LPVSLicense} entity with the specified SPDX identifier.
      */
-    @Query(
-            value =
-                    "SELECT * FROM license_list WHERE license_list.license_spdx = :spdxId ORDER BY id DESC LIMIT 1",
-            nativeQuery = true)
+    @Query("SELECT l FROM LPVSLicense l WHERE l.spdxId = :spdxId ORDER BY l.id DESC LIMIT 1")
```

Comment:
as an option to simplify more, [Repository Method Definition](https://docs.spring.io/spring-data/jpa/reference/repositories/query-methods-details.html) can be used:

eg: findTopBySpdxIdOrderByIdDesc

Reply:
We will plan such refactoring in the future. Thanks for the suggestion.